app/configs/crashesListViewTable.ts

```diff
@@ -172,7 +172,7 @@ const crashesListViewfilterCards: FilterGroup[] = [
           },
         ],
       },
-      // todo: "Other" sitch
+      // todo: "Other" switch
```

Comment:
i have no good excuse for not implementing this yet.  but it's on the todo list!